ConnectionJustForToday/ConnectionJustForToday/Models/JFTReading.swift

```diff
@@ -6,30 +6,67 @@
 //
 
 import Foundation
+import CryptoKit
```

Comment:
Where are you using CryptoKit?

Reply:
Removed. I added when we were talking about a way for the posts to be dynamic.